thirtybees/installation/migration.md

```diff
@@ -8,6 +8,20 @@ permalink: /installation/migration/
 ---
 
 # Thirty Bees - Migration from Prestashop 1.6 to Thirty Bees
+## Using Módule version 1.0.1 
+The migration is more easy, you can migrate direct from PS 1.6 to TB 1.0.6 direct  
```

Comment:
This `o` in 'Module' doesn't look like an English character.

Reply:
ok...in Brazil is 'Módulo' with accent...sorry.